togglwrapper/api.py

```diff
@@ -153,7 +153,8 @@ def get(self, id=None, start_date=None, end_date=None):
 
     def start(self, data):
         """ Starts a new time entry. """
-        return super(TimeEntries, self).update(child_uri='/start', data=data)
+        uri = self._compile_uri(child_uri='/start')
+        return self.toggl.post(uri, data=data)
```

Comment:
This works! It's a little outside the pattern established by the other parts of the class, though.

We could tweak `CreateMixin` to accept the same format that `.update` took, in order to avoid calling "private" functions here - it's what I'd prefer.

I don't mind trying to whip up my own PR tonight if you're time constrained on coming back to this - I'm aware it's been a while since you've opened the PR 😄